test/tsrc/test.c

```diff
@@ -41,10 +41,15 @@
 #include "string_util_test.h"
 #include "validated_move_test.h"
 #include "wasm_api_test.h"
+#include "wmp_test.h"
+#include "wmp_maker_test.h"
 #include "word_prune_test.h"
 #include "word_test.h"
 
 void run_all(void) {
+  test_wmp();
```

Comment:
Make sure we run the full test before merging.

Reply:
oops my bad, fixed. i still haven't figured out how to detect leaks locally so i've been relying heavily on github ci, and it was taking too long to run the whole set of sets, so that's what i was up to